pkg/controller/installation/bootstrapReconciler.go

```diff
@@ -132,6 +142,44 @@ func (r *Reconciler) checkCloudResourcesConfig(ctx context.Context, serverClient
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
+func (r *Reconciler) checkRateLimitsConfig(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	rlConfig := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      marin3rconfig.RateLimitConfigMapName,
+			Namespace: r.installation.Namespace,
+		},
+	}
+
+	if _, err := controllerutil.CreateOrUpdate(ctx, serverClient, rlConfig, func() error {
+		if rlConfig.Data == nil {
+			rlConfig.Data = map[string]string{}
+		}
+
+		if _, ok := rlConfig.Data["rate_limit"]; ok {
+			return nil
+		}
+
+		defaultConfig := map[string]*marin3rconfig.RateLimitConfig{
+			marin3rconfig.ManagedApiServiceSKU: {
+				Unit:            "minute",
+				RequestsPerUnit: 231,
```

Comment:
These values will most likely change in the near future depending on load testing results and BU discussions but the default should try to closely match what we want to set in production. The current goal is 20 million requests per day which equals 231 requests per second or 13,860 requests per minute.

To stay consistent across the board we should probably stick with `minute` as the unit and `13860` as the `RequestsPerUnit` value.

Reply:
I would only expect the configuration to be set here in development scenarios, as it won't override an existing configuration (which should have been created already with the add-on), shouldn't we then have low values as you suggested in the earlier comment?
I will update the value in the MR in managed-tenants to `13860` though

Reply:
My understanding was that should the configmap not exist on cluster this would be the default that would get set? Not sure if it could ever happen but if Hive didn't apply the configmap for some reason it might be safer to have these defaults set to production values? The dev configmap map added as part of this PR should also override these right?

Reply:
That's a good point. The dev configmap won't be created on an add-on flow, so if hive doesn't reconcile it there wouldn't be a config map at all. I'll update this value for the production-like one